Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextDoubleTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using Moq;
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextDoubleTests
+    {
+        [Fact]
+        public void NextDouble_ShouldReturnDouble()
```

Comment:
Missing test cases:
- What happens if Random is null?
- NextDouble(this Random random, double minValue, double maxValue) - what happens if minValue is greater than maxValue?
- What happens if the minValue or maxValue are Infinity? And NaN?